Makefile

```diff
@@ -40,9 +37,6 @@ SKYDIVE_PATH:=$(SKYDIVE_PKG)/src/$(SKYDIVE_GITHUB)/
 SKYDIVE_GITHUB_VERSION:=$(SKYDIVE_GITHUB)/version.Version=${VERSION}
 GO_BINDATA_GITHUB:=github.com/jteeuwen/go-bindata/go-bindata
 PROTOC_GEN_GO_GITHUB:=github.com/golang/protobuf/protoc-gen-go
-FLOW_PROTO_FILES=flow/flow.proto flow/set.proto flow/request.proto
-FILTERS_PROTO_FILES=filters/filters.proto
```

Comment:
don't remove these, they are used in SKYDIVE_PROTO_FILES
you should add _easyjson.go in SKYDIVE_PROTO_FILES too
currently "make dist localdist" failed

Reply:
done